docs/v0.5.0/CONTRIBUTING.md

```diff
@@ -45,6 +45,8 @@ The workflow of modifying the Caliper code-base usually consists of the followin
 
 ### Bootstrapping the Caliper repository
 
+> __Note:__ If you are developing from the latest main branch, refer to the instructions in the [vNext contributing guide](https://hyperledger.github.io/caliper/vNext/contributing/). The following instructions apply only apply if you have checked out the v0.5.0 tag and are developing from there.
```

Comment:
the link reference ties us to hyperledger-github.io which we don't really want. If you look in https://github.com/hyperledger/caliper/blob/gh-pages/docs/v0.5.0/Fabric_Configuration_New.md you should see examples of how to link to other parts of the documentation.